ComponentTextKit/CKTextComponent.mm

```diff
@@ -99,11 +99,13 @@ - (CKComponentLayout)computeLayoutThatFits:(CKSizeRange)constrainedSize
 
 - (CK::Component::MountResult)mountInContext:(const CK::Component::MountContext &)context
                                         size:(const CGSize)size
+                         alignmentRectInsets:(UIEdgeInsets)alignmentRectInsets
                                     children:(std::shared_ptr<const std::vector<CKComponentLayoutChild>>)children
                               supercomponent:(CKComponent *)supercomponent
 {
   CK::Component::MountResult result = [super mountInContext:context
                                                        size:size
+                                        alignmentRectInsets:alignmentRectInsets
```

Comment:
Is there any reason this doesn't just take the layout?
I wasn't sure if that would be breaking some sort of abstraction.